test/stub/rest.go

```diff
@@ -0,0 +1,47 @@
+package stub
+
+import (
+	"github.com/go-resty/resty/v2"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+const (
```

Comment:
Am I missing it or are these unused?

Reply:
they are used in the unit tests in other packages ..... all three are used 

I just confirmed in my Goland Jetbrains, which in addition to its usage search, has a feature that flags unused const/vars/funcs.